scripts/test.py

```diff
@@ -42,21 +42,20 @@ def monte_carlo_sampling_chaospy(N_FEATURES, SAMPLES, DISTRIBUTION, rule="latin_
     import chaospy
     from scipy.stats import qmc 
     from sklearn.preprocessing import MinMaxScaler
-
-
+    params = tuple(DISTRIBUTION_PARAMS)
     # Generate a Nfeatures-dimensional latin hypercube varying between 0 and 1:
-    N_FEATURES = f"chaospy.{DISTRIBUTION}(0, 1), "*N_FEATURES
+    N_FEATURES = f"chaospy.{DISTRIBUTION}{tuple(params)}, "*N_FEATURES
     cube = eval(f"chaospy.J({N_FEATURES})")  # writes Nfeatures times the chaospy.uniform... command)
     lh = cube.sample(SAMPLES, rule=rule, seed=seed).T
 
     # rescale distribution to 0-1 if it is not a uniform distribution
     if DISTRIBUTION == "Uniform":
         pass
-    elif DISTRIBUTION == "Normal":
-        mm = MinMaxScaler()
+    elif DISTRIBUTION in ["Normal", "LogNormal", "Triangle", "Beta", "Gamma"]:
+        mm = MinMaxScaler(feature_range=(0,0.99))
```

Comment:
Would be perfect to understand why does this `0.0000000002` appear when using `feature_range=(0,1)`. Any ideas on that? May it have anything to do with numeric precision?

Reply:
I think it is as you have suspected. The issue is most likely a numerical precision issue.

Reply:
To handle the precision issue, I added an extra argument to clip the values within the feature range.

Reply:
Nice approach!